ADR/xxxx-kite-poc-architecture-and-components.md

```diff
@@ -12,6 +12,121 @@ The [Konflux Issue Tracking Engine](https://github.com/konflux-ci/kite) (KITE) i
+
+## Primary Use Cases
+**1. Developer Morning Check-in Workflow**
+
+**Scenario**: A developer is managing 8 components across 3 different applications.
+
+**Current Problem**:
+- That developer must manually check each component's build and test pipelinerun statuses across multiple views.
+- Each view only displays information related to a specific application or component. This leads to more time spent gathering data.
+
+
+**With Issues Dashboard**:
+- The developer opens the Konflux Issues Dashboard
+- They have a summary of all existing issues related to their applications, components, etc
+- Rather than having to dig through each issue individually across multiple views and then decide what to work on first, issues can be filtered by severity.
+  - This allows them to work on critical issues first, potentially unblocking them from moving forward.
+
+**Value**:
+- Time saved (e.g. 5-30 minutes daily)
+- Immediate awareness of all blocking issues
+
+
+**2. Cross-Component Failure Correlation**
+
+**Scenario**: A developer notices that his application is failing but doesn't realize it's related to a shared library issue affecting multiple teams.
+
+**Current Problem**: Multiple teams independently debug the same root cause, wasting collective hours.
+
+**With Issues Dashboard**:
+- Single issue: "Shared library v2.1 causing build failures"
+- **Scope**: Components affected by the same issue can fall under the same [scope](https://github.com/konflux-ci/kite/blob/main/packages/backend/docs/Webhooks.md#issue-grouping-with-scope-objects).
+- **Grouping**: By grouping the issues, multiple teams are alerted of the same error, preventing separate debugging efforts.
+**Auto-resolution**: When the library is fixed, all related issues can be [resolved automatically](https://github.com/konflux-ci/kite/blob/main/packages/backend/docs/Webhooks.md#automatic-issue-resolution-via-custom-webhooks) because of the shared scope.
+
+**3. User Support Rotation**
+
+**Scenario**: I am an associate on a Konflux Support rotation. A user complains that their application won't build and they don't know why
+
+**Current Problem**:
+- Depending on the supporting associates skill set, they may or may not have the knowledge of how to investigate such issues.
+  - Querying a Konflux cluster and checking the logs for a failing pipelinerun can require multiple skills that are out of reach for a supporting associate.
+- The user might have to navigate through multiple views in order to get a clear picture of what could be the cause.
+- A power user capable of investigating logs for a CR on a Konflux cluster still needs to figure out where to check.
+
+**With Issues Dashboard**:
+- A non-technical associate can point to the Issues Dashboard and point them towards three issues related to failed builds.
+  - Each issue has logs to those builds, allowing for deeper investigation.
```

Comment:
It is also possible to show only failed pipelines and click on them and see the logs without KITE

Reply:
| failed releases, failed builds

Those are already visible in the UI
 
| MintMaker fails

As I mentioned in the other comments, I think that the problem is that mintmaker doesn't run the dependency update pipelines in the user's namespace. In addition, if the is any configuration issue, it can report it through the status section of the component resource.